### PR TITLE
Merge webui publish fix + bump ipfs-api version

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

**Comment by diasdavid** [comment]
@krl can you merge this and push the new `webui` to `go-ipfs`? 

//cc @jbenet @whyrusleeping

**Comment by whyrusleeping** [comment]
:+1: :fireworks: :smile: :+1: wooo!

File: package.json [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+  "version": "0.3.0",
   "description": "A frontend for IPFS",
-  "main": "js/main.jsx",
   "dependencies": {
     "bootstrap": "3.3.4",
     "d3": "3.5.5",
     "debug": "2.1.3",
     "font-awesome": "4.3.0",
-    "ipfs-api": "1.1.4",
+    "ipfs-api": "2.1.5",
```
Comment (jbenet) [inline package.json]: might want to bump to 2.1.6 if https://github.com/ipfs/node-ipfs-api/pull/45 merges first

**Comment by jbenet** [comment]
LGTM. @diasdavid i'm good to ship this.

the tests... arent testing.

**Comment by diasdavid** [comment]
@jbenet done. Tests fail because the repo doesn't have any tests :(

**Pull request merged**